scripts/gen-edpm-kustomize.sh

```diff
@@ -0,0 +1,202 @@
+        edpm_ovn_metadata_agent_default_bind_host: ${EDPM_OVN_METADATA_AGENT_BIND_HOST}
+        edpm_chrony_ntp_servers:
+        - ${EDPM_CHRONY_NTP_SERVER}
+
+        # 99-standalone-vars
+        ctlplane_dns_nameservers:
+        - 192.168.122.1
+        dns_search_domains: []
+        edpm_ovn_dbs:
+        - ${EDPM_OVN_DBS}
+
+        registry_name: quay.io
+        registry_namespace: tripleozedcentos9
+        image_tag: current-tripleo
+        edpm_ovn_controller_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-ovn-controller:{{ image_tag }}"
+        edpm_iscsid_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-iscsid:{{ image_tag }}"
+        edpm_logrotate_crond_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-cron:{{ image_tag }}"
+        edpm_nova_compute_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-compute:{{ image_tag }}"
+        edpm_nova_libvirt_container_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-nova-libvirt:{{ image_tag }}"
+        edpm_ovn_metadata_agent_image: "{{ registry_name }}/{{ registry_namespace }}/openstack-neutron-metadata-agent-ovn:{{ image_tag }}"
+
+        gather_facts: false
+        enable_debug: false
+        # edpm firewall, change the allowed CIDR if needed
+        edpm_sshd_configure_firewall: true
+        edpm_sshd_allowed_ranges: ${EDPM_SSHD_ALLOWED_RANGES}
+        # SELinux module
+        edpm_selinux_mode: enforcing
+        edpm_hosts_entries_undercloud_hosts_entries: []
+        # edpm_hosts_entries role
+        edpm_hosts_entries_extra_hosts_entries:
+        - 172.17.0.80 glance-internal.openstack.svc neutron-internal.openstack.svc cinder-internal.openstack.svc nova-internal.openstack.svc placement-internal.openstack.svc keystone-internal.openstack.svc
+        - 172.17.0.85 rabbitmq.openstack.svc
+        - 172.17.0.86 rabbitmq-cell1.openstack.svc
+        edpm_hosts_entries_vip_hosts_entries: []
+        hosts_entries: []
+        hosts_entry: []
+        plan: overcloud
+    - op: replace
+      path: /spec/roles/edpm-compute/nodeTemplate/ansibleSSHPrivateKeySecret
+      value: ${EDPM_ANSIBLE_SECRET}
+EOF
+if [ "$EDPM_SINGLE_NODE" == "true" ]; then
+cat <<EOF >>kustomization.yaml
+    - op: remove
+      path: /spec/nodes/edpm-compute-1
+EOF
+else
+cat <<EOF >>kustomization.yaml
+    - op: replace
+      path: /spec/nodes/edpm-compute-1/ansibleHost
+      value: ${EDPM_COMPUTE_1_IP}
+    - op: replace
+      path: /spec/nodes/edpm-compute-1/openStackAnsibleEERunnerImage
+      value: ${OPENSTACK_RUNNER_IMG}
+    - op: replace
+      path: /spec/nodes/edpm-compute-1/node/ansibleSSHPrivateKeySecret
+      value: ${EDPM_ANSIBLE_SECRET}
+    - op: replace
+      path: /spec/nodes/edpm-compute-1/node/ansibleVars
+      value: |
+        ctlplane_ip: ${EDPM_COMPUTE_1_IP}
+        internal_api_ip: 172.17.0.100
+        storage_ip: 172.18.0.100
+        tenant_ip: 172.10.0.100
```

Comment:
these are conflicting with the edpm-compute-0 settings as that uses the same IPs. I think we need to make these dynamic as well and using the same postfix (101 by default) as the EDPM_COMPUTE_1_IP

Reply:
thank you! I've pushed the postfix change